vcpkg/overlay/qgis/portfile.cmake

```diff
@@ -1,5 +1,5 @@
-set(QGIS_REF 78789804b5edf89777fe06b3fbeb88a8ef8f05e2)
-set(QGIS_SHA512 3035289511ea864e667930d9d17c38e50aed9f72e6065804ff01ca5a5de91b0be15be7238d1354241600578ce901373a09804eb50dadcfc0a0727df32f9a42bf)
+set(QGIS_REF 650a203e269aaa285dde8e37623a0c5129f094b1)
```

Comment:
Is there a tag name (for readability)?

Reply:
That's current master. Not planning to settle on this revision, merely wanted to see if the settings asan error was still there.